linting/code-quality/code_quality_test.m

```diff
@@ -0,0 +1,78 @@
+% Linter warning prototype
+
+classdef code_quality_test < matlab.unittest.TestCase
+
+  methods
+
+    function serious_problems = filterProblems(testCase, problems, mfile)
+      % Some problems should only be warnings, not errors.
+      % This function filters them out.
+      % You can add more problems to the list if you want.
```

Comment:
Could you improve this comment to note whether warnings fail CI or not?

Reply:
Good observation! Does it look good now?